docs/source/get_started/computers.rst

```diff
@@ -145,15 +145,12 @@ The configuration of computers happens in two steps.
    .. tip:: You can press ``<CTRL>+C`` at any moment to abort the setup process.
      Nothing will be stored in the DB.
    
-   .. note:: For multiline inputs (like the prepend text and the append text, see below)
-     you have to press ``<CTRL>+D`` to complete the input, even if you do not want
-     any text.
    
    Here is a list of what is asked, together with an explanation.
    
-   * **Computer name**: the (user-friendly) name of the new computer instance 
+   * **Computer label**: the (user-friendly) name of the new computer instance 
```

Comment:
Could you please replace "name" by "label" in the text below as well?
Just to avoid confusion between name / label